aviary/examples/run_aviary_example.py

```diff
@@ -0,0 +1,127 @@
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_range": False,
+            "initial_mach": (0.2, "unitless"),
+            "final_mach": (0.72, "unitless"),
+            "mach_bounds": ((0.18, 0.74), "unitless"),
+            "initial_altitude": (0.0, "ft"),
+            "final_altitude": (32000.0, "ft"),
+            "altitude_bounds": ((0.0, 34000.0), "ft"),
+            "throttle_enforcement": "path_constraint",
+            "fix_initial": True,
+            "constrain_final": False,
+            "fix_duration": False,
+            "initial_bounds": ((0.0, 0.0), "min"),
+            "duration_bounds": ((64.0, 192.0), "min"),
+        },
+        "initial_guesses": {"times": ([0, 128], "min")},
+    },
+    "cruise": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "optimize_mach": False,
+            "optimize_altitude": False,
+            "polynomial_control_order": 1,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_range": False,
+            "initial_mach": (0.72, "unitless"),
+            "final_mach": (0.72, "unitless"),
+            "mach_bounds": ((0.7, 0.74), "unitless"),
+            "initial_altitude": (32000.0, "ft"),
+            "final_altitude": (34000.0, "ft"),
+            "altitude_bounds": ((23000.0, 38000.0), "ft"),
+            "throttle_enforcement": "boundary_constraint",
+            "fix_initial": False,
+            "constrain_final": False,
+            "fix_duration": False,
+            "initial_bounds": ((64.0, 192.0), "min"),
+            "duration_bounds": ((56.5, 169.5), "min"),
+        },
+        "initial_guesses": {"times": ([128, 113], "min")},
+    },
+    "descent": {
+        "subsystem_options": {"core_aerodynamics": {"method": "computed"}},
+        "user_options": {
+            "optimize_mach": False,
+            "optimize_altitude": False,
+            "polynomial_control_order": 1,
+            "num_segments": 5,
+            "order": 3,
+            "solve_for_range": False,
+            "initial_mach": (0.72, "unitless"),
+            "final_mach": (0.36, "unitless"),
+            "mach_bounds": ((0.34, 0.74), "unitless"),
+            "initial_altitude": (34000.0, "ft"),
+            "final_altitude": (500.0, "ft"),
+            "altitude_bounds": ((0.0, 38000.0), "ft"),
+            "throttle_enforcement": "path_constraint",
+            "fix_initial": False,
+            "constrain_final": True,
+            "fix_duration": False,
+            "initial_bounds": ((120.5, 361.5), "min"),
+            "duration_bounds": ((29.0, 87.0), "min"),
+        },
+        "initial_guesses": {"times": ([241, 58], "min")},
+    },
+    "post_mission": {
+        "include_landing": False,
+        "constrain_range": True,
+        "target_range": (1906, "nmi"),
+    },
+}
+
+
+prob = av.AviaryProblem(phase_info, "simple", "FLOPS", av.AnalysisScheme.COLLOCATION)
```

Comment:
analysis_scheme is a keyword argument, so we should use the keyword here (particularly since it is a user-facing example.